examples/with-yarn-workspaces/packages/web-app/next.config.js

```diff
@@ -0,0 +1,6 @@
+const withTM = require('next-plugin-transpile-modules')
+
+module.exports = withTM({
+  // Tell webpack to compile the "bar" package
+  transpileModules: ['bar']
```

Comment:
I believe this has to be more specific than just `bar`, but I'm not entirely sure 🤔

Reply:
[next-plugin-transpile-modules](https://github.com/KeitIG/next-plugin-transpile-modules/blob/master/index.js#L4-L5) will convert it to an `include` and `exclude` map.

```js
const includes = [/bar(?!.*node_modules)/]
const exclude = [/node_modules(?!\/bar(?!.*node_modules))/]
```

Or do you mean I should rename the entire packages to something like `my-react-component` or `left-pad` to make it easier to understand? If yes do you have some funny names 😁?